pkg/asset/installconfig/ovirt/validaton.go

```diff
@@ -125,3 +127,12 @@ func validateVNICProfile(platform ovirt.Platform, con *ovirtsdk.Connection) erro
 	}
 	return nil
 }
+
+func validURL(val interface{}) error {
+	uri, ok := val.(string)
+	if !ok {
+		return fmt.Errorf("cannot check url validity on type %v", reflect.TypeOf(val).Name())
```

Comment:
```suggestion
		return fmt.Errorf("cannot check url validity on type %T", val)
```
https://golang.org/pkg/fmt/

Reply:
Thx for the suggestion.